docs/section21/Rule21-11.md

```diff
@@ -0,0 +1,38 @@
+
+# Boiler - Rule 21-11  
+
+**Rule ID:** 21-11  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled as primary only with continuous variable flow and a minimum turndown of 25%.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+
+## Rule Logic:  
+
+- For each fluid loop in B_RMR: `for fluid_loop_b in B_RMR.ASHRAE229.fluid_loops:`
+
+  - Check if fluid loop is heating type: `if fluid_loop_b.type == "HEATING":`
+
+    **Rule Assertion:**
+
+    - Case 1: For heating hot water loop, if loop is modeled as primary only with continuous variable flow and a minimum turndown of 25%: `if ( NOT fluid_loop_b.child_loops ) AND ( fluid_loop_b.heating_design_and_control.flow_control == "VARIABLE_FLOW" ) AND ( fluid_loop_b.heating_design_and_control.minimum_flow_fraction == 0.25 ): PASS`
+
```

Comment:
@mkarpman do you recommend splitting this to two rules? I kept them to be the same because they're interrelated- but can split this to two rules if appropriate.

Reply:
I would either split it into three rules, that (1) the loop is primary-only, (2) the flow is continuous variable and (3) the minimum flow fraction is 25%, or provide one pass and three fail outcomes. I think splitting this into three separate rules will be cleaner.

Reply:
@mkarpman makes sense. @xingcx I have updated the HW TCD spreadsheet to have 3 rules - 21-11A, 21-11B, 21-11C.
these numbers are temporary for now, we will correct them after the RDS have been approved and merged into develop. Please do split this RDS into 3 RDS as suggested by Maria above.

Reply:
Rule 21-11 is now split into 3 rules. This PR will be updated to reflect Rule 21-11A.